ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -182,12 +183,19 @@ public class RouterConfig {
   public final int routerConnectionCheckoutTimeoutMs;
 
   /**
-   * Timeout for requests issued by the router to the network layer.
+   * Timeout for requests waiting at the router layer.
    */
   @Config(ROUTER_REQUEST_TIMEOUT_MS)
-  @Default("2000")
+  @Default("4000")
   public final int routerRequestTimeoutMs;
```

Comment:
We should set a very high default time for this (maybe 1 hour?). Its possible for an account to issue a ttlupdate for a composite blob, and the last of those chunk requests to be stuck in quota queue for long because the account doesn't have sufficient quota.